src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -117,7 +168,9 @@ const ChatItemMyTextV2 = ({
                 />
 
                 <View style={{flexDirection: 'row'}}>
-                  <Text style={messageStyle(true, messageType)}>{message}</Text>
+                  {message?.trim() !== '' && (
+                    <Text style={messageStyle(true, messageType)}>{message}</Text>
+                  )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition `message?.trim() !== ''` is added to prevent rendering an empty text component when the message is an empty string. This could be a result of messages with only attachments and no text.
```diff
+                   {message?.trim() !== '' && (
+                     <Text style={messageStyle(true, messageType)}>{message}</Text>
+                   )}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->